### DISCUSSION TITLE
Failed to create StressChaos on a kubernetes cluster using containerd

### DISCUSSION THREAD

**Discussion by daydayupyjw** [chaos-mesh/chaos-mesh#3722] [Q&A]
## Bug Report

**What version of Kubernetes are you using?**

v1.24.6
**What version of Chaos Mesh are you using?**

v2.4.1
**What did you do? / Minimal Reproducible Example**

Follow the instructions to create a StrssChaos
**What did you expect to see?**
Experiment has started
**What did you see instead?**
Failed to apply chaos: rpc error: code = Unknown desc = expected docker:// but got container
**Output of chaosctl**

Failed to apply chaos: rpc error: code = Unknown desc = expected docker:// but got container

**Comment by STRRL** [comment]
If you are using containerd as the Kubernetes Cluster CRI, please setup helm chart values like: `helm install chaos-mesh chaos-mesh/chaos-mesh -n=chaos-mesh --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock --version 2.4.1`


https://chaos-mesh.org/docs/production-installation-using-helm/#step-4-install-chaos-mesh-in-different-environments
